### PR TITLE
Updated responses for Icelandic language.

### PR DESCRIPTION
**Pull request by samueljon**
Updating Icelandic responses.

### CODE REVIEW

File: responses/is/HassClimateGetTemperature.yaml [hunk 1]
```diff
@@ -2,4 +2,4 @@ language: is
 responses:
   intents:
     HassClimateGetTemperature:
-      default: '{{ state.state }} degrees'
+      default: '{{ state.state }} gráður'
```
Comment (schizza) [inline responses/is/HassClimateGetTemperature.yaml:5]: update `'` to `"` in senteces

File: responses/is/HassClimateSetTemperature.yaml [hunk 2]
```diff
@@ -2,4 +2,4 @@ language: is
 responses:
   intents:
     HassClimateSetTemperature:
-      default: Temperature set to {{ slots.temperature }} degrees
+      default: Setti hitastig til {{ slots.temperature }} gráður
```
Comment (schizza) [inline responses/is/HassClimateSetTemperature.yaml:5]: sentences should be quoted with `"`

**Review by schizza — CHANGES_REQUESTED** [review]
all sentences should be quoted with `"`

**Review by schizza — APPROVED** [review]
Looks good to me. As I cannot evaluate correct grammar

**Comment by samueljon** [comment]
Thanks, @schizza 

I have been translating various open-source projects into Icelandic for decades now. Home Assistant is one of them. Looking forward to see this merged ;-)

**Pull request merged**